snapcraft/plugins/go.py

```diff
@@ -181,6 +190,10 @@ def _run(self, cmd, **kwargs):
         env = self._build_environment()
         return self.run(cmd, cwd=self._gopath_src, env=env, **kwargs)
 
+    def _run_output(self, cmd, **kwargs):
```

Comment:
I would rather change this for all plugins in one sweep as this follows the pattern we've been using started by @kyrofa. A bug would come in handy :wink: